src/hooks/useIndexOfLastVisibleChild.mdx

```diff
@@ -35,30 +35,40 @@ of the children until they exceed the width of the container.
   const children = useMemo(() => {
     const indexOfOverflowStart = indexOfLastVisibleChild + 1;
     const childrenList = elements.map((element, i) => (
-      <div className="px-4 pt-2" style={i >= indexOfOverflowStart ? invisibleStyles : {}}>{element}</div>
+      <div
```

Comment:
all the changes in this file are just a bit of clean up, had a lot of warnings in console about `key` prop and other props stuff (like missing props etc.)